docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -54,7 +54,8 @@ Feathr will get the configurations in the following order:
 | JDBC_TABLE                                            | If using JDBC endpoint as offline store, this config specifies the JDBC table to read from                                                                                                                                                                                         | Required if using JDBC sources as offline store                           |
 | JDBC_USER                                             | If using JDBC endpoint as offline store, this config specifies the JDBC user                                                                                                                                                                                                       | Required if using JDBC sources as offline store                           |
 | JDBC_PASSWORD                                         | If using JDBC endpoint as offline store, this config specifies the JDBC password                                                                                                                                                                                                   | Required if using JDBC sources as offline store                           |
-| KAFKA_SASL_JAAS_CONFIG                                | If using EventHub as a streaming input source, this configures the KAFKA stream. If using EventHub, read [here](https://github.com/Azure/azure-event-hubs-for-kafka#updating-your-kafka-client-configuration) for how to get this string from the existing string in Azure Portal. | Required if using Kafka/EventHub as streaming source input.               |
+| KAFKA_SASL_JAAS_CONFIG                                | see [here](#kafkasasljaasconfig) for more details.
```

Comment:
Seems should be (#kafka_sasl_jaas_config)

Reply:
thanks, fixed